infinite_tracing/lib/infinite_tracing/config.rb

```diff
@@ -64,7 +64,7 @@ def trace_observer_host
       # is overridden by the presence of the port on the host entry.
       def port_from_host_entry
         port_str = NewRelic::Agent.config[:'infinite_tracing.trace_observer.host'].scan(%r{:(\d+)$}).flatten
-        if port = (port_str[0] and port_str[0].to_i)
+        if port = (port_str[0]&.to_i)
```

Comment:
These parens can go now that we don't have the `and` anymore.

Reply:
Thank you! Resolved in: 45e17184fb95e9d79d458bd33fd5486eefdc392c